string color
bool start
---
bool status